src/lib/getCommitsFiltered.ts

```diff
@@ -50,7 +50,6 @@ export default async function getCommitsFiltered(
   cwd: string,
   dir: string,
   lastRelease?: string,
-  nextRelease?: string,
```

Comment:
This value was filled with `context.nextValue` in the AnalyzeCommit step, however `context.nextValue` is only set **after** this step.
Moreover, I don't see any use case where semantic-release would run while already having a nextVersion 🤔 

What do you think @GuillaumeRahbari ?